ang/civicase/activity/card/directives/activity-card.directive.js

```diff
@@ -187,12 +186,17 @@
      */
     function getCaseDetailUrl () {
       var caseTypeId = $scope.activity.case.case_type_id;
-      var caseType = caseTypes[caseTypeId];
+      var caseType = CaseType.getById(caseTypeId);
       var caseTypeCategory = caseTypeCategories[caseType.case_type_category];
       var caseDetailUrl = 'civicrm/case/a/?' +
         $.param({ case_type_category: caseTypeCategory.name }) +
         '#/case/list';
-      var angularParams = $.param({ caseId: $scope.activity.case_id });
+      var angularParams = $.param({
+        caseId: $scope.activity.case_id,
+        cf: JSON.stringify({
```

Comment:
Why do we need to stringify this `cf` key only?

Reply:
If we don't pass the `cf` as a string it will pass the filters as this:
`?caseId=7&cf[case_type_id.is_active]=0&cf={"case_type_category":"awards"}`
and we want them to be passed like this:
`?case_type_category=awards#/case/list?caseId=7&cf={"case_type_id.is_active":"0","case_type_category":"awards"}`

We only need the filter to make disabled cases visible. We could pass other filters, such as the case type, but that's not necessary as it doesn't follow the existing behaviour which is to display all cases without filters, but display the case details for the given ID.